src/Strings.php

```diff
@@ -0,0 +1,28 @@
+<?php namespace Lean\Utils;
+
+/**
+ * A suite of functions for working strings.
+ *
+ * Class Strings.
+ *
+ * @package Lean\Utils
+ */
+class Strings
```

Comment:
Yes used with no namespace it's but in this case is under a namespace you can have a class named `Date` or `MySQLi` as long as it's mapped inside of a namespace that is not global is all good.